### DISCUSSION TITLE
set Timezone

### DISCUSSION THREAD

**Discussion by cbram** [hassio-addons/addon-influxdb#346] [Q&A] [answered]
Hi, is there a way to set the Timezone for the InfluxDB? I think in docker this would be en env variable...

Thanks

**Comment by frenck** [comment, marked as answer]
> Hi, is there a way to set the Timezone for the InfluxDB? I think in docker this would be en env variable...

Nope, timezone setting is managed by Home Assistant, not the add-on.

../Frenck
